### ISSUE TITLE
Get-Buildvariable overwrites CommitMessage on switch default 

### ISSUE THREAD

**Issue by jfishe** [RamblingCookieMonster/BuildHelpers#126]
https://github.com/RamblingCookieMonster/BuildHelpers/blob/f385bd8b68bf49d24938fdea3fdec3eb83858817/BuildHelpers/Public/Get-BuildVariable.ps1#L246 sets CommitMessage instead of CommitHash.

From the context, I think the following was intended:

```PowerShell 
if(-not $CommitHash)
    {
        if($WeCanGit)
        {
            $CommitHash = Invoke-Git @IGParams -Arguments "log --format=%H -n 1"
        }        
    }
```

**Comment by RamblingCookieMonster** [comment]
Yikes, yes, that.  Or move the wecangit conditional into `default` for the switch. thanks for catching this!!  fancy a PR?

**Comment by RamblingCookieMonster** [comment]
FIxed in 2.0.13!  Ouch.  Thanks again for the heads up : )
